README.md

```diff
@@ -1 +1,38 @@
-# Playground
\ No newline at end of file
+# Playground
+
+
```

Comment:
🔪

Reply:
What do you think to add a little explanation about the project here! For more instructions talk to @alvesisaque